### PR TITLE
Added Clear button

### PR DESCRIPTION
**Pull request by jimmy-huang**
Added clear button to clear canvas, also I deleted the build directory and fixed a build dependency issue on MacOS.

### CODE REVIEW

File: src/toolbar.js [hunk 1]
```diff
@@ -305,6 +314,7 @@ export class Toolbar extends LitElement {
         <div class="width-title">Line Width</div>
         <mwc-slider pin markers step="1" value="8" min="1" max="20" id="line-width-slider" @change="${this._lineWidthChanged}"></mwc-slider>
       </div>
+      <mwc-button slot="action" raised id="usi-clear-button">Clear</mwc-button>
```
Comment (darktears) [inline src/toolbar.js:317]: let's just call it clear-button. The demo works everywhere and that piece is not usi-specific :).

Comment (jimmy-huang) [reply src/toolbar.js]: updated patch

File: src/toolbar.js [hunk 2]
```diff
@@ -227,11 +227,13 @@ export class Toolbar extends LitElement {
   firstUpdated() {
     this._usiColorCell = this.shadowRoot.querySelector('#usi-read-color-cell');
     this._snackbar = this.shadowRoot.querySelector('#snackbar');
+    this._usiClearButton = this.shadowRoot.querySelector('#clear-button');
     this._usiReadButton = this.shadowRoot.querySelector('#usi-read-button');
     this._drawingPreferencesCheckbox = this.shadowRoot.querySelector('#drawing-preferences-checkbox');
     this._predictedEventsCheckbox = this.shadowRoot.querySelector('#predicted-events-checkbox');
     this._predictedEventsHighlightCheckbox = this.shadowRoot.querySelector('#predicted-events-highlight-checkbox');
     this._lineWidthSlider = this.shadowRoot.querySelector('#line-width-slider');
+    this._usiClearButton.onpointerdown = this._clearCanvas.bind(this);
```
[the remark this answers is not in the document]
Comment (jimmy-huang) [reply src/toolbar.js]: fixed

File: src/toolbar.js [hunk 3]
```diff
@@ -227,11 +227,13 @@ export class Toolbar extends LitElement {
   firstUpdated() {
     this._usiColorCell = this.shadowRoot.querySelector('#usi-read-color-cell');
     this._snackbar = this.shadowRoot.querySelector('#snackbar');
+    this._usiClearButton = this.shadowRoot.querySelector('#clear-button');
```
[the remark this answers is not in the document]
Comment (jimmy-huang) [reply src/toolbar.js]: fixed

**Pull request merged**